public/js/components/SourceFooter.js

```diff
@@ -96,7 +96,7 @@ const SourceFooter = React.createClass({
         (!isEnabled("prettyPrint")
           && !isEnabled("blackBox")
           && !isEnabled("search"))) {
-      return null;
+      return dom.div({ className: "source-footer" });
```

Comment:
This is going to show a blank footer for now while none of the features are enabled, which might look sort of weird. Is there any way to avoid showing the footer until it's actually used for something?